### DISCUSSION TITLE
Tooltips on Android

### DISCUSSION THREAD

**Discussion by IanRawley** [AvaloniaUI/Avalonia#16073] [Q&A]
I've been experimenting with ReactiveUI.Validation and Avalonia recently, and while everything works nicely on desktop, I'm having trouble on Android.  Specifically, how is a user on an Android device using a touch display supposed to pop up a ToolTip?  A long press didn't work, and obviously there isn't a pointer so we can't pointer over.  Am I missing something?

**Comment by NickBarton** [comment]
I had the same problem, as a work-around, I'm using ContextMenu with binding to the prompt with a GotFocus event to show the "menu". 

In the code behind:

    private void TextBox_GotFocus(object sender, GotFocusEventArgs e)
    {
        field.ContextMenu?.Open();
    }

where field is defined in the TextBox as x:Name="field".


On Android, shows the "menu" at the top of the screen which is helpful.

**Comment by ahopper** [comment]
Just testing on v12 and long-press tooltips don't appear either on android, windows (surface) or linux (rpi touch screen).  Is there a plan for this?
